docs/egress.md

```diff
@@ -79,6 +80,9 @@ spec:
         role: web
   egressIP: 10.10.0.8 # can be populated by Antrea after assigning an IP from the pool below
   externalIPPool: prod-external-ip-pool
+  rateLimit:
```

Comment:
IMO we should remove this here and move it to the TrafficShaping section, at least while it is still "Alpha"

Reply:
Thanks. Done.